packages/domain/lib/model/palidrome_response.dart

```diff
@@ -0,0 +1,9 @@
+class PalidromeResponse {
+  final bool isPalidrome;
```

Comment:
fyi - usually fields in response are made optional. what if u'll receive the following response?

{
    "isPalindrome": null
}

Reply:
1. Ok, Is it right way to handle this null in the usecase ? 
2. Or it will be better handling it in the bloc ? 

PS: know i chose first way : 
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/99803326/159901424-5278800c-b057-4203-bdf4-4156c0be05c7.png">

Reply:
it's ok. as a variant, u can also do in a such way: `final isPalindrome = response.isPalindrome ?? false;`